lib/instances/TextButton.lua

```diff
@@ -24,4 +26,22 @@ TextButton.properties.TextSize = InstanceProperty.typed("number", {
 	end,
 })
 
+TextButton.properties.TextWrapped = InstanceProperty.typed("boolean", {
+	getDefault = function()
+		return false
+	end,
+})
+
+TextButton.properties.TextXAlignment = InstanceProperty.typed("number", {
+	getDefault = function()
+		return TextXAlignment.Left
+	end,
+})
+
+TextButton.properties.TextYAlignment = InstanceProperty.typed("number", {
+	getDefault = function()
+		return TextYAlignment.Top
+	end,
+})
```

Comment:
Make sure you switch these to use the enum helper for `InstanceProperty` instead of `number` or else you won't be able to assign to them.